systemd/coreos-installer-secure-ipl-reboot.service

```
@@ -0,0 +1,14 @@
+[Unit]
+Description=Reboot from zVM LOADDEV after CoreOS Installer
```

The unit is designed to handle the LPAR case, so we might as well reflect that in its description. E.g.

```suggestion
Description=Reboot in Secure IPL mode after CoreOS Installer
```